Classes/ViewHelpers/Data/ValueForKeyViewHelper.php

```diff
@@ -37,27 +36,32 @@
  */
 class ValueForKeyViewHelper extends AbstractViewHelper
 {
+
+    /**
+     * @return void
+     */
+    public function initializeArguments()
+    {
+        parent::initializeArguments();
+        $this->registerArgument('array', 'array', 'The array to extract the value from', true);
+        $this->registerArgument('key', 'string', 'The key to extract the value for', true);
+    }
+
     /**
-     * @param array  $array The array to extract the value from
-     * @param string $key   The key to extract the value for
-     *
      * @return string|int|bool|array
      */
-    public function render($array, $key)
+    public function render()
     {
         return self::renderStatic(
-            [
-                'key' => $key,
-                'array' => $array,
-            ],
+            $this->arguments,
             $this->buildRenderChildrenClosure(),
             $this->renderingContext
         );
     }
 
     /**
-     * @param array                     $arguments
-     * @param \Closure                  $renderChildrenClosure
+     * @param array $arguments
```

Comment:
These PHPDoc blocks may be removed safely, as they are already contained in the method's signature

Reply:
done